Severity statistics and csv export

csordasmarton opened the pull request:
- Showing severity report count at the statistics page.
- Statistics can be exported to CSV.

![cc_sev_statistics](https://user-images.githubusercontent.com/6695818/32664219-e459c59e-c630-11e7-90c1-3089a5ea26fe.png)

Closes #1104

whisperity commented:
The button feels weird for me. It could be that the alignment of the text is off a few pixels, or the colour is too saturated.

csordasmarton commented:
@whisperity You have good eyes to catch pixel alignment problems :smile: I have fixed this and I have also changed the color of the export button:

![cc_statistics_export](https://user-images.githubusercontent.com/6695818/32716518-9e8cefe6-c856-11e7-9cdb-dd0555661ead.png)

What do you think?

whisperity commented on www/scripts/codecheckerviewer/CheckerStatistics.js:
```diff
@@ -336,6 +475,25 @@ function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
+
+      dom.create('a', {
+        class : 'export-to export-to-csv',
+        innerHTML : 'Export: CSV',
```
Looking good now. I think we could omit the `:` here.

whisperity commented:
I think this feature (checker stats) is `CodeChecker cmd sum` which does have an `--output <format>` option.
